docs/source/en/package_reference/inference_endpoints.md

```diff
@@ -0,0 +1,46 @@
+# Inference Endpoints
+
+Inference Endpoints offers a secure production solution to easily deploy any `transformers`, `sentence-transformers` and `diffuser` models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
```

Comment:
```suggestion
Inference Endpoints offers a secure production solution to easily deploy any 🤗 Transformers, sentence-transformers, and 🤗 Diffusers models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
```

Reply:
FYI, I actually copy-pasted the introduction from https://huggingface.co/docs/inference-endpoints/index. Personal feeling, but I preferred to get rid of all the :hugs: as I feel they slow down the reading without bringing much (there were 7 :hugs: in the initial version :confused:)
So I'd prefer to keep it as it is if that is fine with you.

```suggestion
Inference Endpoints offers a secure production solution to easily deploy any `transformers`, `sentence-transformers`, and `diffusers` models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
```